### PR TITLE
Enable user to turn off/on logging from sqlalchemy

### PR DESCRIPTION
**Pull request by NickSchimek**
Sometimes the logging is useful, but other times it just gets in the
way. Enable the user to turn off/on logging as they desire from their
`.env` file.

For example when tests fail you'll often see large output from this logger, which can be helpful sometimes, but most of the time you just want to see the failure message.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -55,8 +56,10 @@ def check_if_token_in_blocklist(jwt_header, decrypted_token):
     def format_unauthorized_message(message):
         return {app.config["JWT_ERROR_MESSAGE_KEY"]: message.capitalize()}, 401
 
-    logging.basicConfig()
-    logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+    if os.getenv("DB_LOGGING"):
```
Comment (TNorbury) [inline app.py]: Would it be better to do something like:
```suggestion
    if os.getenv("DB_LOGGING") == "ON":
```

Just thinking that some people (namely me), might try to turn logging off by setting the env variable to "OFF"

**Review by TNorbury — DISMISSED** [review]
Looks good!

File: app.py (hunk 1 above)
Comment (aedwardg) [reply app.py]: This is a fair point. I can definitely see people changing this to "OFF" later instead of commenting it out.

**Review by aedwardg — DISMISSED** [review]

File: app.py (hunk 1 above)
Comment (NickSchimek) [reply app.py]: Yes, thank you!

**Pull request merged**